frontend/public/locales/fr/translation.json

```diff
@@ -533,6 +533,7 @@
     "invite": "Inviter",
     "send": "Envoyer",
     "fields": "Champs",
+    "upload": "Télécharger",
```

Comment:
I think that "télécharger" has a better meaning with the the download process not with the upload process.
I propose to use "Téléverser" as a french translation for the upload operation
![image](https://github.com/user-attachments/assets/8ac45563-0cb5-42d8-9c43-69d553c90721)
https://support.google.com/googleplay/answer/11012086?hl=fr-CA&co=GENIE.Platform%3DDesktop